general/g.version/main.c

```diff
@@ -118,7 +118,7 @@ int main(int argc, char *argv[])
 	fprintf(stdout, "version=%s\n", GRASS_VERSION_NUMBER);
 	fprintf(stdout, "date=%s\n", GRASS_VERSION_DATE);
 	fprintf(stdout, "revision=%s\n", GRASS_VERSION_GIT);
-	fprintf(stdout, "build_date=%d-%02d-%02d\n", YEAR, MONTH, DAY);
+	fprintf(stdout, "build_date=%d-%02d-%02dT%s\n", YEAR, MONTH, DAY, __TIME__);
```

Comment:
Why do we need `__TIME__`? Does `build_date` here mean the time of `g.version` built, not the entire GRASS package? Also, this timestamp doesn't have the timezone offset. `2020-04-04T10:40:00` vs. `2020-04-04T10:40:00-04:00` if we want the same format for dates. I don't think at least the GNU C compiler has `__TIMEZONE__` or similar... if we want the ISO format here, the only option is to pass the timezone offset from `configure` and do effectively `gcc -D__TIMEZONE__=$(date +%z)`. However, again, I believe that `build_date` should be the build timestamp for the entire package. Maybe, configure time makes more sense here?

Reply:
I agree, why do we need `__TIME__` for build_date? If we need it, it should go to `g.version/local_proto.h` and be made available as TIME, just like YEAR, MONTH, and DAY.

Reply:
I agree that the code should be improved. The question is if we need to change `build_date` at all. This change was inspired by https://github.com/OSGeo/grass/pull/325#issuecomment-607511540. I will remove it from this PR. 

Small overview `g.version`:

```
date=2020 # A YEAR
build_date=2020-04-19 # A DATE
libgis_date=2020-04-08T22:36:27+02:00 # A TIMETAMP
```

Newly (see #405)

```
-- config date
2020-04-08T22:36:27+02:00 # A TIMETAMP
```

Reply:
Removed by 7c5e3e9b641f58edaa06e1f02efe3750efdfb16b